Kitodo-LongTermPreservationValidation/src/main/java/org/kitodo/longtermpreservationvalidation/jhove/KitodoJhoveNisoImageMetadataHelper.java

```diff
@@ -0,0 +1,523 @@
+    );
+
+    /**
+     * Resolves a label by following the index through an index and label array.
+     * 
+     * @param value the index
+     * @param labelArray the label array
+     * @param indexArray the index array
+     * @return the label
+     */
+    private static Optional<String> parseLabelFromInt(int value, String[] labelArray, int[] indexArray) {
+        if (Objects.nonNull(value) && value != NisoImageMetadata.NULL) {
+            int index = ArrayUtils.indexOf(indexArray, value);
+            if (index != ArrayUtils.INDEX_NOT_FOUND) {
+                return Optional.of(labelArray[index]);
+            }
+            return Optional.of(String.valueOf(value));
+        }
+        return Optional.empty();
+    }
+
+    /**
+     * Extract indexed label properties and add them to the metadata map.
+     * 
+     * @param metadata the niso image matadata 
+     * @param map the simple string map that is being filled
+     */
+    private static void extractIndexedLabelProperties(NisoImageMetadata metadata, Map<String, String> map) {
+        // indexed label properties
+        for (Map.Entry<String, Function<NisoImageMetadata, Integer>> entry : INDEXED_LABEL_PROPERTIES_GETTER_MAP.entrySet()) {
+            Integer value = entry.getValue().apply(metadata);
+            if (Objects.nonNull(value) && value != NisoImageMetadata.NULL) {
+                String[] labelArray = INDEXED_LABEL_PROPERTIES_LABEL_ARRAY_MAP.get(entry.getKey());
+                int[] indexArray = INDEXED_LABEL_PROPERTIES_INDEX_ARRAY_MAP.get(entry.getKey());
+                Optional<String> label = parseLabelFromInt(value, labelArray, indexArray);
+                if (label.isPresent()) {
+                    map.put(entry.getKey(), label.get());
+                }
+            }
+        }
+    }
+
+    /**
+     * Extract single label properties from niso image metadata and add them to the metadata map.
+     * 
+     * @param metadata the niso image metadata
+     * @param map the simple string map that is being filled
+     */
+    private static void extractSingleLabelProperties(NisoImageMetadata metadata, Map<String, String> map) {
+        // label properties
+        for (Map.Entry<String, Function<NisoImageMetadata, Integer>> entry : SINGLE_LABEL_PROPERTIES_GETTER_MAP.entrySet()) {
+            Integer value = entry.getValue().apply(metadata);
+            if (Objects.nonNull(value) && value != NisoImageMetadata.NULL) {
+                String[] labelArray = LABEL_PROPERTIES_LABEL_ARRAY_MAP.get(entry.getKey());
+                if (value >= 0 && value < labelArray.length) {
+                    map.put(entry.getKey(), labelArray[value]);
+                }
+            }
+        }
+    }
+
+    /**
+     * Extract multiple label properties from niso image metadata and add them to the metadata map.
+     * 
+     * @param metadata the niso image metadata
+     * @param map the simple string map that is being filled
```

Comment:
The corresponding changes apply downwards, so: JavaDoc’s first sentence should be a verbal phrase (third form singular), NISO to be written in capital letters ...

These changes can be made in a later pull request, but it would be nice to have it straightened out.